packages/ui/src/organisms/ProductCard/ProductCard.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { forwardRef } from 'react'
+import type { HTMLAttributes } from 'react'
+
+export interface ProductCardProps
+  extends Omit<HTMLAttributes<HTMLDivElement>, 'role'> {
```

Comment:
I think you forgot to remove this `role`, since you are not using on this component.